src/main/scala/com/stratio/example/RabbitMQConsumer.scala

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
Please, move this file to src/**test**/scala/com/stratio/example/RabbitMQConsumer.scala. We don't want it to be packaged